webapp/src/pages/Settings.tsx

```diff
@@ -31,48 +82,625 @@ const Settings: React.FC = () => {
+                Object.entries(
+                  resultingConfig[customizationConfig] ??
+                    CONFIG_SUB_FIELDS[customizationConfig] ??
+                    {}
+                )
+              ).map(
+                ([field, value], index) =>
+                  !ANALYSES_CUSTOMIZATION_IGNORE_FIELDS.includes(field) && (
+                    <Box
+                      key={index}
+                      display="flex"
+                      flexDirection="row"
+                      justifyContent="flex-start"
+                      marginLeft={2}
+                      gap={5}
+                    >
+                      {displayNumberField(customizationConfig, field, value)}
+                    </Box>
+                  )
+              )}
+            </Box>
+            {divider}
+          </Box>
+        )}
+      </Box>
+    ));
+
+  return (
+    <Box sx={{ height: "100%" }}>
+      <Paper
+        variant="outlined"
+        sx={{ height: "90%", padding: 2, overflow: "auto" }}
+      >
+        <Typography variant="subtitle1" marginBottom={3}>
+          View and edit certain fields from your config file. Once your changes
+          are saved, expect some delays for recomputing the affected tasks.
+        </Typography>
+        <AccordionLayout
+          name="Project Configurations"
+          description="contains mandatory fields that specify the dataset to load in Azimuth"
+          link="reference/configuration/project/"
+        >
+          {getProjectConfigSection()}
+        </AccordionLayout>
+        <AccordionLayout
+          name="Model Contract Configurations"
+          description="defines how Azimuth interacts with the ML pipelines and the metrics"
+          link="reference/configuration/model_contract/"
+        >
+          {getModelContractConfigSection()}
+        </AccordionLayout>
+        <AccordionLayout
+          name="Analyses Customization"
+          description="four analyses configured in Azimuth"
+          link="reference/configuration/analyses/"
+        >
+          {getAnalysesCustomization()}
+        </AccordionLayout>
+      </Paper>
+      <Box
+        sx={{
+          height: "10%",
+          display: "flex",
+          justifyContent: "space-between",
+          alignItems: "center",
+          paddingY: 2,
+        }}
+      >
+        <Button variant="contained" onClick={() => setPartialConfig({})}>
+          Discard
+        </Button>
+        <Box
+          display="flex"
+          flexDirection="row"
+          justifyContent="flex-end"
+          alignItems="center"
+          gap={2}
+        >
+          {FIELDS_TRIGGERING_STARTUP_TASKS.some((f) => partialConfig[f]) && (
+            <FormHelperText>
+              <Warning
+                color="primary"
+                sx={{
+                  position: "relative",
+                  top: 3,
+                  right: 0.5,
+                  color: (theme) => theme.palette.warning.contrastText,
```

Comment:
I would suggest setting `color="warning"` and not overwriting the `color` in `sx`.
```suggestion
                color="warning"
                sx={{
                  position: "relative",
                  top: 3,
                  right: 0.5,
```